core/src/test/java/io/grpc/internal/ServerImplTest.java

```diff
@@ -112,10 +111,11 @@ public static void beforeStartUp() {
   }
 
   private ExecutorService executor = Executors.newSingleThreadExecutor();
-  private MutableHandlerRegistry registry = new MutableHandlerRegistryImpl();
+  private InternalHandlerRegistry registry = new InternalHandlerRegistry.Builder().build();
```

Comment:
There should probably be a new test to make sure we do a look up in `registry` before `fallbackRegistry`.